cmd/kubeadm/app/cmd/cmd_test.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestNewKubeadmCommand(t *testing.T) {
```

Comment:
I don't think we really need this test, which has already been covered by tests in [`cobra.Command`](https://github.com/spf13/cobra/blob/master/command_test.go).

Reply:
ok, waiting on confirmation before closing this PR.

Reply:
@kad, @dmmcquay should this be closed?

Reply:
No strong opinion from me, but I think we can close.